README.md

```diff
@@ -50,4 +50,31 @@ We aim to:
 * [Compliance Viewer](https://github.com/18F/compliance-viewer)
 * [The Google Docs folder](https://drive.google.com/a/gsa.gov/folderview?id=0B5fn0WMJaYDnTVctaUgzZm94bnc&usp=sharing) (private)
 
+### Compliance Masonry Related Links and Data
+
+#### Quick Start Examples
+- [cg-application-ssp-example](https://github.com/18F/cg-application-ssp-example): An setup example for applications built on Cloud.Gov.  
+
+#### Tooling
+- [Compliance Masonry CLI](https://github.com/opencontrol/compliance-masonry): CLI tool for building docs.  
+- [OpenControl YAML Editor](https://github.com/opencontrol/OpenControl-YAML-editor).  
+- [doc-template](https://github.com/opencontrol/doc-template): A library that extends golang's template engine to docx.  
+
+#### OpenControl Schemas
+- [OpenControl Schemas](https://github.com/opencontrol/schemas): Schemas for data organized in the OpenControl format.  
+
+#### OpenControl Data  
+- [AWS Compliance](https://github.com/opencontrol/aws-compliance): Component documentation for AWS.  
+- [CloudFoundry Compliance](https://github.com/opencontrol/cf-compliance): Compliance documentation for Cloud Foundry.  
+- [Cloud.Gov Compliance](https://github.com/18F/cg-compliance): Compliance documentation for Cloud Foundry.  
+- [GSA Certifications](https://github.com/18F/GSA-Certifications): Certification documentation for GSA (Includes control requirements for GSA's LATO).  
+- [FedRAMP Certifications](https://github.com/opencontrol/FedRAMP-Certifications): Certification documentation for FedRAMP.  
+- [NIST-800-53 Standard](https://github.com/opencontrol/NIST-800-53-Standards): Standard documentation for NIST-800-53.  
```

Comment:
I just mean so that you don't have to list everything that's under the opencontrol org separately. Up to you.